gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -730,7 +730,7 @@ static gboolean
 gst_tensor_transform_set_option_data (GstTensorTransform * filter)
 {
   gchar *filter_name;
-  gboolean ret = TRUE;
+  gboolean ret = FALSE;
```

Comment:
How about using `filter->loaded` as the return value?

Reply:
Thank you, I found the error case to revert the option and updated this PR.

About your suggestion, plz consider the case if filter-loaded is true (changing option string)
If option is set and trying to update option string, filter->loaded might be true.

Reply:
Understand. Thank you for your kind explanation.